### DISCUSSION TITLE
Website and Documentation Availability

### DISCUSSION THREAD

**Discussion by ertauren** [eclipse-iceoryx/iceoryx#1814] [Q&A] [answered]
Hello, I am trying to reach documentation in website
https://iceoryx.io/
but the website is down. Is there any other source for documentation?

**Comment by dkroenke** [comment, marked as answer]
@ertauren 
Thanks for reporting this, we noticed that too and created a ticket on Eclipse Helpdesk: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2333, it seems to be a DNS resolution issue.
In the meantime you can clone the [iceoryx-web](https://github.com/eclipse-iceoryx/iceoryx-web) containing the website for this project.

We will keep you here posted once the webiste is back online.

**Reply by dkroenke** [reply]
https://iceoryx.io is back online.
